rs/sns/testing/README.md

```diff
@@ -62,7 +61,7 @@ by `sns-testing` identity will be automatically adopted.
 Additionally, `sns-testing-init` will output the ID of the NNS neuron that should be used to create NNS proposals.
 ```
 ...
-Use the following NNS neuron ID for further testing: 1
+Use the following NNS neuron ID for further testing: 13469749498645661119
```

Comment:
Could you give a hint as to where this odd value is coming from?

Reply:
But conceptually, what is special about this neuron? We could at least say in the readme, e.g., "This is the preconfigured neuron controlled by identity X, and it is expected to have a lot of voting power, so when it submits NNS proposals, they get adapted right away." 

I suppose the same explanation would help with neuron ID 1, just that 1 already looks like something pre-configured.

Reply:
https://github.com/dfinity/ic/pull/6474/commits/2774bb35f3cdc2b84f287ae21330ac6b5d1f7716